package.json

```diff
@@ -48,7 +48,7 @@
     "lint-staged": "^10.1.3",
     "lodash.startcase": "^4.4.0",
     "prettier": "1.19.1",
-    "prettier-plugin-solidity": "^1.0.0-beta.1",
+    "prettier-plugin-solidity": "^1.0.0-beta.10",
```

Comment:
I had to bump this version to correctly lint some new solidity 0.8.x code. in particular the `unchecked` syntax showed up as having errors without this.